### ISSUE TITLE
Change the appearance of the line number gutter

### ISSUE THREAD

**Issue by hansom** [jfmoy/Fraise#28]
I would like to be able to change the background color and the font of the line number gutter. As it is now the gutter looks too much like the editor area and isn’t distinguishable enough. Thanks.

**Comment by brhfl** [comment]
Yes, please! Opposite problem - if you change the editor so that it's light on dark, the gutter is far too contrasty to the editor area!
